website/docs/markdown-features.mdx

```diff
@@ -259,4 +258,4 @@ function Clock(props) {
 }
 ```
 
-**Note:** The React Live component is rather big in bundle size. It is an opt-in.
+**Note:** The React Live component is rather big in bundle size, hence it is an opt-in (lazily loaded) by default.
```

Comment:
Its not lazily loaded. Its an opt-in as in it doesnt come by default in preset-classic.

Reply:
How can I make this statement clearer?